src/orderbook.ts

```diff
@@ -179,6 +199,21 @@ function sortOffersAscending(left: Offer, right: Offer) {
   }
 }
 
+function sortOffersDescending(left: Offer, right: Offer) {
+  return sortOffersAscending(left, right) * -1;
+}
+
+function priceToCoverAmount(amount: Fraction, offers: Offer[]) {
```

Comment:
nit: I wouldn't start the name of this function with a `noun`. Use better a verb (like `get` or `calculate`)

Reply:
Thanks to typescript, since it's clear from the signature that the function returns something I don't think adding get/calculate as a prefix adds much value (the question I'm asking myself as a maintainer is what does it return and that thing is a noun).

Reply:
Sorry I disagree, although it's still a nit, and I wouldn't argue too much about it

You can make any search about the topic, people agree on using verbs. Clean code is a reference book, and they are clear on the topic. 

Some literature about it:
https://www.nr.no/directdownload/lexicon.pdf
https://medium.com/coding-skills/clean-code-101-meaningful-names-and-functions-bf450456d90c
http://codelegance.com/semantic-method-naming/